### PR TITLE
Fix/Update translations

### PR DESCRIPTION
**Pull request by NathanBnm**
I fixed the "Reminders" label which wasn't translated. I also fixed some French translations.

### CODE REVIEW

File: src/MainGrid.vala [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class Badger.MainGrid : Gtk.Grid {
         column_spacing = 12;
         orientation = Gtk.Orientation.VERTICAL;
 
-        var heading = new Granite.HeaderLabel (_ ("Reminders"));
+        var heading = new Granite.HeaderLabel (_("Reminders"));
```
Comment (dar5hak) [inline src/MainGrid.vala]:
```suggestion
        var heading = new Granite.HeaderLabel (_ ("Reminders"));
```

So Badger follows the convention of having a space between a method/function name and the opening parenthesis. That applies even if the name is one character, so I'm gonna keep this space.

**Comment by dar5hak** [comment]
Thank you so much, Nathan, for keeping the translations up-to-date. You rock! :dark_sunglasses:

File: src/MainGrid.vala (hunk 1 above)
Comment (dar5hak) [reply src/MainGrid.vala]: Umm, I don't seem to have permission to commit to your branch. I'll just merge it and then change in master.

Comment (NathanBnm) [reply src/MainGrid.vala]: Okay understand, I though this was an error.

**Comment by NathanBnm** [comment]
Thank you too, I really appreciate your idea with this app. And I'm working to make elementary OS and apps the most accesible  as possible :1:

**Pull request merged**